hash/m20.xml

```diff
@@ -31,126 +33,164 @@ license:CC0-1.0
 			</dataarea>
 		</part>
 	</software>
 
 	<software name="pcos13f">
 		<description>Olivetti PCOS 1.3f</description>
-		<year>19??</year>
+		<year>1982</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
-			<dataarea name="flop" size="185194">
+			<dataarea name="flop" size="286720">
 				<rom name="pcos13f.imd" size="185194" crc="61047402" sha1="31c716245911544597981474b69c23a9a3a084a7"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="pcos20f">
+	<software name="pcos20f" supported="no"><!-- Missing FM track0, boot with -ramsize 512k -->
 		<description>Olivetti PCOS 2.0f</description>
-		<year>19??</year>
+		<year>1982</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pcos20f.img" size="286720" crc="87297648" sha1="10c3247d2d72f6d2f06c3b582704db55ed5a1e28"/>
+				<rom name="pcos20f.img" size="286720" crc="87297648" sha1="10c3247d2d72f6d2f06c3b582704db55ed5a1e28" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="pcos30e">
+	<software name="pcos20h" supported="partial"><!-- Missing FM track0 -->
+		<description>Olivetti PCOS 2.0h</description>
+		<year>1982</year>
+		<publisher>Olivetti</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="286720">
+				<rom name="pcos20h.img" size="286720" crc="55f470a7" sha1="7ac52fa8b52954410bc4ac496d6038a9a8204e8f" status="baddump"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcos30e" supported="partial"><!-- Missing FM track0 -->
 		<description>Olivetti PCOS 3.0e</description>
-		<year>19??</year>
+		<year>1983</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="pcos30e.img" size="286720" crc="ee2f9763" sha1="7f7dddb7815d0a4615ed4eac8f4709fb67688b46"/>
+				<rom name="pcos30e.img" size="228963" crc="ee2f9763" sha1="7f7dddb7815d0a4615ed4eac8f4709fb67688b46" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="pcos41a">
+	<software name="pcos30ea" cloneof="pcos30e">
+		<description>Olivetti PCOS 3.0e (Copy 1)</description>
+		<year>1983</year>
+		<publisher>Olivetti</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="286720">
+				<rom name="pcos30e.td0" size="286720" crc="c6de0eec" sha1="2e1eb3d152a92ed1262899208ef298179e1b61b3"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcos30eb" cloneof="pcos30e">
+		<description>Olivetti PCOS 3.0e (Copy 2)</description>
+		<year>1983</year>
+		<publisher>Olivetti</publisher>
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="286720">
+				<rom name="pcos30e.imd" size="286720" crc="383a2131" sha1="62698a393328dc3ac1cdd5d72360623f741e54e1"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Are “(Copy 1)” and “(Copy 2)” actually parts of the titles, or just for disambiguation?  If it’s just for disambiguation, use “(alt 1)” and “(alt 2)” for consistency with other software lists.